### PR TITLE
Navbar

### PR DESCRIPTION
**Pull request by Riyaz350**
@AbdulAzizFahim created dashboard dropdown menu and some more animation. Fixed laptop responsive issue.

### CODE REVIEW

File: src/Layouts/Home/Home.jsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    return (
+        <div>
+            <NavbarDefault></NavbarDefault>
+            <Outlet></Outlet>
+            
+        </div>
+    );
+};
+
+export default Home;
```
Comment (AbdulAzizFahim) [inline src/Layouts/Home/Home.jsx:12]: Please add new line.

File: src/Layouts/Home/Home.jsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import NavbarDefault from '../../Shared/Navbar/NavbarDefault';
+import { Outlet } from 'react-router-dom';
+
+const Home = () => {
+    return (
+        <div>
+            <NavbarDefault></NavbarDefault>
+            <Outlet></Outlet>
+
```
Comment (AbdulAzizFahim) [inline src/Layouts/Home/Home.jsx]: Remove empty spaces

File: src/Layouts/Home/Lost/Lost.jsx [hunk 3]
```diff
@@ -0,0 +1,9 @@
+
+const Lost = () => {
+    return (
+        <div>
+        </div>
```
Comment (AbdulAzizFahim) [inline src/Layouts/Home/Lost/Lost.jsx]: Don't use empty div, instead use
<>
</>

File: src/Shared/Navbar/NavbarDefault.jsx [hunk 4]
```diff
@@ -103,14 +106,27 @@ import { NavLink } from "react-router-dom";
...
+        <ul className="p-2 shadow menu right-0 w-fit space-y-2 dropdown-content z-[1] bg-base-100 rounded-box lg:w-52">
+        <h1 className="text-[#022b3a] font-bold p-2 pl-4">Username</h1>
+        <NavLink  className={` ${menuClass} hover:bg-[#022b3a] hover:text-white`}>
+        <MdOutlineDashboard />Dashboard
+        </NavLink>
+        <NavLink  className={` ${menuClass} hover:bg-[#022b3a] hover:text-white`}>
+        <IoIosLogOut /> Log Out
+        </NavLink>
+        </ul>
+        {/* <ul className="p-2 shadow menu lg:hidden right-0 w-fit  dropdown-content z-[1] bg-base-100 rounded-box lg:w-52">
```
Comment (AbdulAzizFahim) [inline src/Shared/Navbar/NavbarDefault.jsx]: Don't commit comments unless meaningful

**Review by AbdulAzizFahim — COMMENTED** [review]
Please resolve the comments

**Comment by Riyaz350** [comment]
> Please resolve the comments

resolved

**Review by AbdulAzizFahim — APPROVED** [review]

**Pull request merged**